src/command.coffee

```diff
@@ -66,3 +69,19 @@ class Command
       version.replace(/-.*$/, '')
     else
       version
+
+  loadInstalledAtomMetadata: (callback) ->
+    @getResourcePath (resourcePath) =>
+      try
+        {version, electronVersion} = require(path.join(resourcePath, 'package.json')) ? {}
+        version = @normalizeVersion(version)
+        @installedAtomVersion = version if semver.valid(version)
+
+      @electronVersion = process.env.ATOM_NODE_VERSION ? electronVersion ? '0.22.0'
```

Comment:
If we want to eventually rename this to `ATOM_ELECTRON_VERSION`, what do you think about starting to support that now by adding a case for `process.env.ATOM_ELECTRON_VERSION` before `process.env.ATOM_NODE_VERSION`? This would make it a little easier to deprecate `ATOM_NODE_VERSION` and remove it for apm 2.0.